Sources/Rendering/Core/TextProperty/index.js

```diff
@@ -0,0 +1,64 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkProperty2D from 'vtk.js/Sources/Rendering/Core/Property2D';
+
+// ----------------------------------------------------------------------------
+// vtkTextProperty methods
+// ----------------------------------------------------------------------------
+
+function vtkTextProperty(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkTextProperty');
+}
+
+// ----------------------------------------------------------------------------
+// Object factory
+// ----------------------------------------------------------------------------
+
+const DEFAULT_VALUES = {
+  fontFamily: 'Arial',
+  fontColor: [0, 0, 0],
+  fontStyle: 'normal',
+  backgroundColor: [1, 1, 1],
+  fontSizeScale: (resolution) => resolution / 1.8,
+  resolution: 200,
+  shadowColor: [1, 1, 0],
```

Comment:
does not seem like a good default value.

I would make it `undefined`:
`// shadowColor: [1, 1, 1],`